MXNet nightly test fails often during last two weeks

## Description
The MXNet nightly test is failing often during the last month.

## To Reproduce

see the logs of the last runs of MXNet
https://github.com/awslabs/gluon-ts/actions?query=branch%3Amaster+workflow%3A%22MXNet+nightly%22

## Error message or code output

There seem to be two errors, one is that the package `wheel` is not installed. 
The other seems to be a missing file

### example of missing package & missing file
```bash
Run python -m pip install --upgrade pip
Requirement already up-to-date: pip in /opt/hostedtoolcache/Python/3.6.10/x64/lib/python3.6/site-packages (20.1)
Could not build wheels for pip, since package 'wheel' is not installed.
/home/runner/work/_temp/db171c80-a970-4ff0-8547-ed33b6933564.sh: line 2: 2: No such file or directory
##[error]Process completed with exit code 1.
```
[link](https://github.com/awslabs/gluon-ts/runs/665301384?check_suite_focus=true)

### example of missing file error

```bash
Run python -m pip install --upgrade pip
Collecting pip
Downloading pip-20.1-py2.py3-none-any.whl (1.5 MB)
Installing collected packages: pip
Attempting uninstall: pip
Found existing installation: pip 20.0.2
Uninstalling pip-20.0.2:
Successfully uninstalled pip-20.0.2
Successfully installed pip-20.1
/home/runner/work/_temp/67ef72f6-ca20-47eb-b746-22f61df7ba5a.sh: line 2: 2: No such file or directory
##[error]Process completed with exit code 1.
```
For missing file error see [here](https://github.com/awslabs/gluon-ts/runs/658020193?check_suite_focus=true), [here](https://github.com/awslabs/gluon-ts/runs/651383544?check_suite_focus=true) & [here](https://github.com/awslabs/gluon-ts/runs/654875037?check_suite_focus=true).

## Environment
- Operating system: Ubuntu
- Python version: 3.6 & 3.7

--- a/.github/workflows/mxnet_nightly.yml
+++ b/.github/workflows/mxnet_nightly.yml
@@ -1,10 +1,11 @@
 name: MXNet nightly
 
 on:
-  schedule:
+  push
+  #schedule:
     # * is a special character in YAML so you have to quote this string
     # nightly:
-    - cron:  '0 0 * * *'
+    #- cron:  '0 0 * * *'
 
 jobs:
   build:
@@ -27,7 +28,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install --pre mxnet<2 -f https://dist.mxnet.io/python
+        pip install --pre "mxnet<2" -f https://dist.mxnet.io/python
         pip install -e ".[shell]"
         pip install -r requirements/requirements-test.txt
         pip install -r requirements/requirements-extras-sagemaker-sdk.txt

--- a/.github/workflows/mxnet_nightly.yml
+++ b/.github/workflows/mxnet_nightly.yml
@@ -1,11 +1,10 @@
 name: MXNet nightly
 
 on:
-  push
-  #schedule:
+  schedule:
     # * is a special character in YAML so you have to quote this string
     # nightly:
-    #- cron:  '0 0 * * *'
+    - cron:  '0 0 * * *'
 
 jobs:
   build:

